src/Crowdin.Api/StringCorrections/IStringCorrectionsApiExecutor.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Crowdin.Api.StringCorrections
+{
+    public interface IStringCorrectionsApiExecutor
+    {
+        Task<ResponseList<Correction>> ListCorrections(
+            int projectId,
+            int stringId,
```

Comment:
same here and to the end of this PR

Only IDs, other `int` properties/params could be left untouched